_drafts/Firmware-encryption-with-python.md

```diff
@@ -0,0 +1,680 @@
+---
+# This is a template for Interrupt posts. Use previous, more recent posts from the _posts/
+# directory for more inspiration and patterns.
+#
+# When submitting a post for publishing, submit a PR with the post in the _drafts/ directory
+# and it will be moved to _posts/ on the date of publish.
+#
+# e.g.
+# $ cp _drafts/_template.md _drafts/my_post.md
+#
+# It will now show up in the front page when running Jekyll locally.
+
+title: "Firmware Encryption with Python"
```

Comment:
Think a bit more about this title I wonder if we can spice it up a bit. Maybe something like this:

```
Encrypting Firmware Updates With Python
```

Reply:
No problem. I like the title you suggest
Let me know if there is anything else thanks